src/configure.js

```diff
@@ -270,6 +270,12 @@ export default function configure(options) {
 				included: true,
 				served: true,
 			},
+			{
+				pattern: path.resolve(__dirname, './lib/jest-globals.js'),
```

Comment:
The files in the `lib` folder are not transpiled or inlined. So this will silently skip loading of this file. Moving it to the parent folder fixes it.